docs/migration-guides/qiskit-2.0.mdx

```diff
@@ -0,0 +1,260 @@
+
+4. The `basis_gates` argument no longer accepts custom basis gates. The suggested alternative
+  is the `target` argument, which can be built from scratch or through [`Target.from_configuration`](/api/qiskit/qiskit.transpiler.Target#from_configuration) 
+  such as:
+
+    ```python
+    from qiskit.circuit.library import XGate
+    from qiskit.transpiler.target import Target
+
+    basis_gates = ["my_x", "cx"]
+    custom_name_mapping = {"my_x": XGate()}
+    target = Target.from_configuration(
+        basis_gates=basis_gates, num_qubits=2, custom_name_mapping=custom_name_mapping
+    )
+    pm = generate_preset_pass_manager(target=target)
+    ```
+
+5. The input combination where a custom `coupling_map` is provided with either a `backend` or `basis_gates`
+  that contain gates with 3 or more qubits is no longer allowed. The coupling map
+  does not provide the necessary connectivity details to be able to determine the  action of the gate.
+  In these cases, the recommended migration path is the creation of a custom target from scratch.
+
+6. The `inst_map` argument has been removed without alternative. It was used to provide pulse information
+  to the transpiler, and the full `qiskit.pulse` module has been removed.
+
+
+<span id="qiskit.pulse"></span>
+## qiskit.pulse
+
+The module `qiskit.pulse` has been completely removed from Qiskit, without replacement.
+This includes the related items:
+
+- the [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit), [`DAGCircuit`](/api/qiskit/qiskit.dagcircuit.DAGCircuit) and [`DAGDependency`](/ap/qiskit/qiskit.dagcircuit.DAGDependency) property `calibrations`.
+- the [`QuantumCircuit`](/api/qiskit/qiskit.circuit.QuantumCircuit) methods `has_calibration_for` and `add_calibration`.
+- the [`DAGCircuit`](/api/qiskit/qiskit.dagcircuit.DAGCircuit) method `has_calibration_for`.
+- the transpiler passes `PulseGates`, `RXCalibrationBuilder`, `RZXCalibrationBuilder`, `EchoRZXWeylDecomposition`, `ValidatePulseGates`.
+- the module `qiskit.scheduler` and all its contents.
+- the [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2) methods `instruction_schedule_map`, `drive_channel`, `measure_channel`, `acquire_channel` and `control_channel`.
+- the [`Target`](/api/qiskit/qiskit.transpiler.Target) methods `has_calibration`, `get_calibration`, `instruction_schedule_map` and `update_from_instruction_schedule_map`.
+- the `inst_map` argument of [`generate_preset_pass_manager`](/api/qiskit/qiskit.transpiler.generate_preset_pass_manager), [`transpile`](/api/qiskit/qiskit.compiler.transpile), and [`Target.from_configuration`](/api/qiskit/qiskit.transpiler.Target#qiskit.transpiler.Target.from_configuration).
+- the `calibration` field of [`InstructionProperties`](/api/qiskit/qiskit.transpiler.InstructionProperties).
+
+This happened because IBM Quantum devices ceased to offer pulse-level access, and the Qiskit team no longer had resources to maintain the little used package.
```

Comment:
```suggestion
This happened because IBM Quantum&reg; QPUs ceased to offer pulse-level access, and the Qiskit team no longer had resources to maintain the little-used package.
```